test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -972,17 +968,13 @@ def test_fftconvolve_broadcast(self, x_shape, y_shape):
             (0, F.fftconvolve, (4, 3, 1, 2), (2, 2, 2)),
             (0, F.fftconvolve, (1, ), (10, 4)),
             (0, F.fftconvolve, (1, ), (2, 2, 2)),
-            # incompatible shape except the last dim
-            (1, F.convolve, (5, 2, 3), (5, 3, 3)),
-            (1, F.convolve, (5, 2, 3), (5, 3, 4)),
-            (1, F.convolve, (5, 2, 3), (5, 3, 5)),
+            # non-broadcastable leading dimensions
+            (2, F.convolve, (5, 2, 3), (5, 3, 3)),
+            (2, F.convolve, (5, 2, 3), (5, 3, 4)),
+            (2, F.convolve, (5, 2, 3), (5, 3, 5)),
             (2, F.fftconvolve, (5, 2, 3), (5, 3, 3)),
             (2, F.fftconvolve, (5, 2, 3), (5, 3, 4)),
             (2, F.fftconvolve, (5, 2, 3), (5, 3, 5)),
```

Comment:
The current case `1` no longer exists, so can we remove the message for case 1 and migrate the what is case=2 now to case=1?